assets/cms/js/tree.js

```diff
@@ -75,6 +75,12 @@ ConcreteTree.prototype = {
             ajaxData.allowFolderSelection = 1
         }
 
+        if (CCM_CID !== undefined) {
```

Comment:
This will raise an error if `CCM_CID` isn't declared which is possible in this context. Use `typeof foo === 'undefined'` instead
```suggestion
        if (typeof CCM_CID !== 'undefined') {
```